code/modules/food&drinks/food/snacks_meat.dm

```diff
@@ -220,3 +220,28 @@
 	bonus_reagents = list("nutriment" = 1, "capsaicin" = 4, "vitamin" = 4)
 	list_reagents = list("nutriment" = 6, "capsaicin" = 5)
 	filling_color = "#FA8072"
+
+/obj/item/weapon/reagent_containers/food/snacks/nugget
+	name = "chicken nugget"
+	filling_color = "#B22222"
+	bonus_reagents = list("nutriment" = 1, "vitamin" = 1)
+	list_reagents = list("nutriment" = 2)
+
+/obj/item/weapon/reagent_containers/food/snacks/nugget/New()
```

Comment:
When you do New(), you gotta call the parent with 

..() 

before the rest of this